recipes/bloaty/all/conanfile.py

```diff
@@ -0,0 +1,81 @@
+import os
+
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+
+class BloatyConan(ConanFile):
+    name = "bloaty"
+    description = """Bloaty McBloatface: a size profiler for binaries"""
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/google/bloaty"
+    license = "Apache-2.0"
+    topics = ("conan", "bloaty", "profiler", "size")
+
+    settings = "os", "arch", "compiler", "build_type"
+
+    requires = (
+        "capstone/4.0.2",
+        "protobuf/3.17.1",
+        "pkgconf/1.7.4",
```

Comment:
Why `pkgconf` is in` requires` and not in `build_requirements()`?

By the way, I advice to always declare dependencies in `requirements()` and not in `requires()`.

Reply:
Moved to `build_requires`.